functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
Actually the schedules created by `New-DbaAgentSchedule` (which requires a Job param to associate the schedule with) are returned by the call to `JobServer.SharedSchedules` so it seems that they're being created as shared schedules anyway, even if only one job references them. Digging further into it, I'm not sure if it's possible to create a schedule that _can't_ be shared.